website/docs/pact_nirvana/step_3.md

```diff
@@ -23,3 +20,42 @@ A good candidate for starting with Pact will:
 3. Verify the pact against the provider \(again, consult the documentation for your chosen language\)
 4. Talk about it.
 5. Decide whether or not Pact is the right tool for you, taking into account that there are situations for which Pact is [a good solution](/faq/#what-is-pact-good-for), and situations for which Pact is [not a good solution](/faq/#what-is-pact-not-good-for).
+
+
+### Write a consumer test with no provider state
+
+Pact is a consumer-driven contract testing framework. That means that the contract is established by the consumer, 
+based on their understanding of the interface they're consuming from the provider and their specific needs as a consumer.  
+This is actually important and powerful. Too often a provider guesses at what their consumer needs and comes up with an 
+API based on that.   When you let the consumers drive the contract, it is much better aligned with their needs.
+ 
+So you write the consumer test first.  You can read more about consumer tests here.
```

Comment:
`here` needs a link.... Or get rid of the "You can read about" sentence

Reply:
Agreed, I had added a todo, as it came from your PR, happy to remove

https://github.com/pact-foundation/docs.pact.io/pull/126/files#diff-461ddc38b008355eccba1ebf69a38a590d9626cab44f5b5c9de5b058a44375cdR45

Reply:
Oh ok I missed that.  Your call whether you want to link or remove

Reply:
We should have a decent page to link to, I will look in the docs, if not will pull it for now and we can add in additional detail later

Reply:
I will link to this page https://docs.pact.io/consumer